### PR TITLE
Add CORS support for AWS Lambda serving

### PR DESCRIPTION
**Pull request by omrihar**
## Description

The DataFrame handler (as well as the JsonHandler) do not return the necessary headers for CORS support in AWS Lambda. I've added to both these handlers a cors keyword (defualts to `"*"`) which, if is not set to None, causes the `handle_aws_lambda_event` to return, along with `"statusCode"` and `"body"` also `headers: {'Access-Control-Allow-Origin': self.cors}`.

## Motivation and Context

This solves a problem I had back in BentoML `0.6.0` up to `0.6.3` where otherwise I was unable to integrate the results from the AWS Lambda deployment with a web-service. I also opened an issue, https://github.com/bentoml/BentoML/issues/509, which was closed due to inactivity (not because the problem was solved). This PR should solve the problem.

## How Has This Been Tested?

I tested this in my own deployment (based on BentoML 0.6.3). Since this is a very minor change, I don't expect this to affect any other area of the codebase. 
I didn't add any tests, since I'm not sure how to test this in your testing environment, but am happy to help.

## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation
- [ ] Test, CI, or build
- [ ] None

## Components (if applicable)
- [ ] BentoService (model packaging, dependency management, handler definition)
- [ ] Model Artifact (model serialization, multi-framework support)
- [ ] Model Server (mico-batching, logging, metrics, tracing, benchmark, OpenAPI)
- [ ] YataiService (model management, deployment automation)
- [ ] Documentation

## Checklist:

- [x] My code follows the bentoml code style, both `./dev/format.sh` and
  `./dev/lint.sh` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] My change requires a change in [bentoml/gallery](https://github.com/bentoml/gallery) example notebooks
- [ ] I have sent a pull request to [bentoml/gallery](https://github.com/bentoml/gallery) to make that change

### CODE REVIEW

**Comment by parano** [comment]
Thanks for the PR @omrihar, LGTM!

cc @bojiang - this might be something we want to add to the new output adapter API, for the user to set common response headers

File: bentoml/handlers/json_handler.py [hunk 1]
```diff
@@ -141,4 +142,10 @@ def handle_aws_lambda_event(self, event, func):
 
         result = func(parsed_json)
         json_output = api_func_result_to_json(result)
+        if self.cors:
+            return {
+                "statsCode": 200,
```
Comment (parano) [inline bentoml/handlers/json_handler.py:147]: `statusCode` typo caught by the CI test

File: bentoml/handlers/json_handler.py [hunk 2]
```diff
@@ -41,8 +41,9 @@ class JsonHandler(BentoHandler):
 
     BATCH_MODE_SUPPORTED = True
 
-    def __init__(self, is_batch_input=False, **base_kwargs):
+    def __init__(self, is_batch_input=False, cors="*", **base_kwargs):
```
Comment (parano) [inline bentoml/handlers/json_handler.py:44]: could you also add the parameter to the docstring above?

File: bentoml/handlers/dataframe_handler.py [hunk 3]
```diff
@@ -193,6 +193,7 @@ def __init__(
         typ="frame",
         input_dtypes=None,
         is_batch_input=True,
+        cors="*",
```
Comment (parano) [inline bentoml/handlers/dataframe_handler.py:196]: could you also add the parameter to the docstring above?

File: bentoml/handlers/json_handler.py (hunk 1 above)
Comment (omrihar) [reply bentoml/handlers/json_handler.py]: fixed

File: bentoml/handlers/dataframe_handler.py (hunk 3 above)
Comment (omrihar) [reply bentoml/handlers/dataframe_handler.py:199]: Added to the docstring. I searched for it on the `__init__` method and didn't notice it's on the class definition.

**Comment by omrihar** [comment]
I added the requested changes

**Comment by parano** [comment]
Thanks for updating the PR @omrihar, merging now!

**Pull request merged**